test/google_crawler/search_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleCrawler.SearchTest do
+  use GoogleCrawler.DataCase
+  alias GoogleCrawler.Search
+  alias GoogleCrawler.Search.Keyword
+
+  describe "list_keywords" do
```

Comment:
let's add function arity here, also in `test/google_crawler_web/controllers/keyword_controller_test.exs` for `index/2`